pkg/opensearch/ppl_response_parser.go

```diff
@@ -17,7 +17,7 @@ type pplResponseParser struct {
 	Response *client.PPLResponse
 }
 
-func newPPLResponseParser(response *client.PPLResponse, query *Query) *pplResponseParser {
+func newPPLResponseParser(response *client.PPLResponse) *pplResponseParser {
```

Comment:
I can change this back, but it was annoying me and my formatter. (this is a private function so its fine to change the funcition signature)